src/clients/humidity.toit

```diff
@@ -5,11 +5,11 @@
 import system.services
 import ..humidity as api
 
-class HumidityService extends services.ServiceClient implements api.HumidityService:
-  static SELECTOR ::= api.SELECTOR
+class HumidityService-v1-0 extends services.ServiceClient implements api.HumidityService-v1-0:
```

Comment:
Why do we need the minor revision here? Do we really need different minor versions to co-exist?

Reply:
as discussed in person.
In theory it would make sense to have minor versions, but for simplicity only using major versions.